Packs/AHA/Integrations/AHA/AHA.py

```diff
@@ -205,13 +263,26 @@ def main() -> None:
             fields = argToList(args.get('fields', ''))
             page = args.get('page', '1')
             per_page = args.get('per_page', '30')
-            command_result = get_features(client, from_date=from_date, feature_name=feature_name, fields=fields, page=page,
-                                          per_page=per_page)
+            command_result = get(client, AHA_TYPE.FEATURES, from_date=from_date, aha_object_name=feature_name,
+                                 fields=fields, page=page, per_page=per_page)
             return_results(command_result)
         elif command == 'aha-edit-feature':
             feature_name = args.get('feature_name', '')
             edit_fields = json.loads(args.get('fields', {}))
-            command_result = edit_feature(client, feature_name=feature_name, fields=edit_fields)
+            command_result = edit(client, aha_type=AHA_TYPE.FEATURES, aha_object_name=feature_name, fields=edit_fields)
+            return_results(command_result)
+        elif command == 'aha-get-ideas':
+            from_date = args.get('from_date', '2020-01-01')
+            idea_name = args.get('idea_name', '')
+            fields = argToList(args.get('fields', ''))
+            page = args.get('page', '1')
+            per_page = args.get('per_page', '30')
+            command_result = get(client, AHA_TYPE.IDEAS, from_date=from_date, aha_object_name=idea_name,
```

Comment:
Let's pass these args to `get()` as `**args` expression, and make `return_results` a one-liner calling `get()`.
Please take into account the following:
1. The default values will be written to the function's signature as default values.
2. the function arg names that don't match the `demisto.args()` names should be switched to the `demisto.args()` so they can be passes as is to the function.